src/mediapipe_internal/mediapipefactory.cpp

```diff
@@ -36,14 +36,17 @@
 
 namespace ovms {
 
+MediapipeFactory::MediapipeFactory(PythonBackend* pythonBackend) {
```

Comment:
Please do not forget about ifdefs for images without python

Reply:
Will add them